src/ovs/aca_ovs_l3_programmer.cpp

```diff
@@ -829,6 +829,107 @@ int ACA_OVS_L3_Programmer::create_or_update_router(RouterConfiguration &current_
             new_routing_rule_entry.next_hop_mac =
                     current_routing_rule.routing_rule_extra_info().next_hop_mac();
 
+            auto remote_host_ip = "";
+            ulong culminative_dataplane_programming_time = 0;
+            for (auto each_neighbor : parsed_struct.neighbor_states()) {
+              NeighborConfiguration current_NeighborConfiguration1 =
```

Comment:
why 'Configuration1' ?

Reply:
> why 'Configuration1' ?

Just following the variable names in the V1 version.